lib/squib/graphics/shapes.rb

```diff
@@ -18,8 +18,15 @@ def rect(box, draw, trans)
     def circle(box, draw)
       x, y, r = box.x, box.y, box.radius
       use_cairo do |cc|
-        cc.move_to(x + r, y)
-        cc.circle(x, y, r)
+        # cc.move_to(x + r, y)
```

Comment:
Should be fine assuming the old functionality doesn't break. Be sure to delete the commented-out code before we merge.